.travis.yml

```diff
@@ -1,6 +1,13 @@
 language: go
 go_import_path: gocloud.dev
-go: "1.11.x"
+go:
+  - "1.11.x"
+  - "1.12.x"
```

Comment:
I'd add a comment here saying something about tests depending on version numbers and search for versions in the test script when bumping these

Reply:
I think it may be clearer to remove the "go" line from here and move it down to "jobs", where we configure the 3 OSs. I think the "matrix" can be removed and handled there too.

Reply:
Done, and I also expanded the matrix for clarity. Not sure there is a way to write this that doesn't repeat either the go version or the list of OSs.